client/src/components/Browse/Browse.styles.js

```diff
@@ -1,13 +1,31 @@
 import React from "react";
 import styled from "@emotion/styled";
-import { mq } from "../../styles/utilities/breakpoints.js";
+import {
+  mq
+} from "../../styles/utilities/breakpoints.js";
+
+export const BrowseContainer = styled.div `
+
 
-export const BrowseContainer = styled.div`
-margin-top: 124px;
-margin: 124px 16px 0;
 
 ${mq[6]} {
-  margin: 124px 0 0;
+  
 
 }
 `;
+
+export const BackgroundImage = styled.div `
+  width: 100%;
+  height: 540px;
+  object-fit: cover;
+  background: linear-gradient(to bottom,
```

Comment:
Adding to Matt's comment, I took a shot at the gradient since I didn't know how to do it in CSS already; Figma allows you to do a lot of stuff really easily!

Here's a codepen I put together with what I got working: https://codepen.io/markbaker/pen/KKVePqb

Reply:
complete